### PR TITLE
Added methods to quickly reply to a message with Markdown or HTML

### PR DESCRIPTION
**Pull request by JosXa**
I had problems mergin urllib3, so this PR closes #725.

@Eldinnie Indentation adjusted as per your request.

Tests are probably gonna have to wait until pytest stops messing up my workflow...

### CODE REVIEW

**Comment by Eldinnie** [comment]
I edited the docstring and merged master. But it's still missing tests for these new methods.
Please add using monkeypatch. See `test_message::test_reply_text` for example

**Comment by Eldinnie** [comment]
@JosXa what is the status for this?

**Comment by JosXa** [comment]
Tests added as discussed

**Comment by JosXa** [comment]
Sorry, I keep committing files from other branches... Fixed now.

File: telegram/message.py [hunk 1]
```diff
@@ -408,6 +408,46 @@ def reply_text(self, *args, **kwargs):
         self._quote(kwargs)
         return self.bot.send_message(self.chat_id, *args, **kwargs)
 
+    def reply_markdown(self, *args, **kwargs):
+        """Shortcut for::
+
+            bot.sendMessage(update.message.chat_id, parse_mode=ParseMode.MARKDOWN, *args, **kwargs)
```
Comment (Eldinnie) [inline telegram/message.py]: Please use the snake_case names in docs. (send_message)

File: telegram/message.py [hunk 2]
```diff
@@ -408,6 +408,46 @@ def reply_text(self, *args, **kwargs):
...
+        kwargs['parse_mode'] = ParseMode.MARKDOWN
+
+        self._quote(kwargs)
+
+        return self.bot.send_message(self.chat_id, *args, **kwargs)
+
+    def reply_html(self, *args, **kwargs):
+        """Shortcut for::
+
+            bot.sendMessage(update.message.chat_id, parse_mode=ParseMode.HTML, *args, **kwargs)
```
Comment (Eldinnie) [inline telegram/message.py]: same as above

File: tests/test_message.py [hunk 3]
```diff
@@ -33,34 +34,52 @@ def message(bot):
 
 @pytest.fixture(scope='function',
                 params=[
-                    {'forward_from': User(99, 'forward_user', False),
-                     'forward_date': datetime.now()},
-                    {'forward_from_chat': Chat(-23, 'channel'),
-                     'forward_from_message_id': 101,
-                     'forward_date': datetime.now()},
+                    {
```
Comment (Eldinnie) [inline tests/test_message.py]: Please reset the dict to the way it was.

**Comment by tsnoam** [comment]
The code itself LGTM.

However, I think it's over-engineering and only bloats the code. Though if the other maintainers find it suitable for ptb, I won't object.

**Comment by tsnoam** [comment]
@jh0ker thinks it a good idea to have this in the library. merging.

**Pull request merged**